js/scripts.js

```diff
@@ -64,15 +64,21 @@
         $(this).removeClass("floating-label-form-group-with-focus");
       });
     });
+    // $(document).ready(function(){
+    //   $(".formopengeneral").click(function(){
+    //     window.open("https://forms.gle/Lc3ByWGvsvuxsadD6");
+    //     console.log("opened form")
+    //   });
+    // });
     $(document).ready(function(){
       $(".formopengeneral").click(function(){
-        window.open("https://forms.gle/Lc3ByWGvsvuxsadD6");
+        window.open("https://docs.google.com/forms/u/3/d/e/1FAIpQLSeIhji_oCPPe4dR_A8rQ3LHOJHZ43PH1ZZIfcXK4S8ua2QpZQ/viewform?usp=send_form");
```

Comment:
You don't have to use Javascript to do this sort of action. 

Inherently in HTML, it supports this via `target='_blank'`: https://www.rapidtables.com/web/html/link/html-link-new-window.html

Reply:
Ok, thanks!